modules/openapi-generator/src/main/resources/typescript/api/exception.mustache

```diff
@@ -8,7 +8,7 @@
  *
  */
 export class ApiException<T> extends Error {
-    public constructor(public code: number, public body: T) {
-        super("HTTP-Code: " + code + "\nMessage: " + JSON.stringify(body))
+    public constructor(public code: number, message: string, public body: T) {
```

Comment:
that's not backwards compatible, is it?

Reply:
Yes there are some slight breaking changes here. (Luckily the generator is still marked as experimental)

1. If you used the constructor in user code, that will have to be changed. (That seems unlikely thought)
2. The `body` property of the exception now really contains the response body. Previously it sometimes contained the message as ApiException<string> and sometimes the body.

All-in-all I feel these are justified breakages.
Maybe it could make sense to put it in v6.0. Is there a timeline on that?